iml-services/iml-corosync/src/main.rs

```diff
@@ -0,0 +1,248 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use futures::TryStreamExt;
+use iml_manager_env::get_pool_limit;
+use iml_postgres::{get_db_pool, sqlx, PgPool};
+use iml_service_queue::service_queue::consume_data;
+use iml_service_queue::service_queue::ImlServiceQueueError;
+use iml_tracing::tracing;
+use iml_wire_types::{
+    high_availability::{Cluster, Node},
+    Fqdn,
+};
+use thiserror::Error;
+
+// Default pool limit if not overridden by POOL_LIMIT
+const DEFAULT_POOL_LIMIT: u32 = 2;
+
+#[tokio::main]
+async fn main() -> Result<(), ImlCorosyncError> {
+    iml_tracing::init();
+
+    let rabbit_pool = iml_rabbit::connect_to_rabbit(1);
+
+    let conn = iml_rabbit::get_conn(rabbit_pool).await?;
+
+    let ch = iml_rabbit::create_channel(&conn).await?;
+
+    let mut s = consume_data::<Cluster>(&ch, "rust_agent_corosync_rx");
+
+    let pool = get_db_pool(get_pool_limit().unwrap_or(DEFAULT_POOL_LIMIT)).await?;
+
+    sqlx::migrate!("../../migrations").run(&pool).await?;
```

Comment:
This is also being done in the device service in my patch. Is there a better place to put this?

Reply:
Not sure it matters, I don't think you can have races with this. If so, we can centralize it later.